### PR TITLE
workgroup production endpoint

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Change to new production endpoint.
- Fix typo in cert names.

# Needed By (Date)
- Sometime in the next few months.

# Urgency
- 1/5 (I rationalize my denominators)

# Steps to Test

1. Check out this branch on either your local or on ACSF dev
2. Install a fresh site with either the Stanford profile or a Jumpstart
3. Log in with SAML and ensure you get your workgroup role mapping.

# Affected Projects or Products
- All of them

# Associated Issues and/or People
- 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by jbickar** [comment]
_Urgency is not a fraction, it’s a rating scale!_

**Comment by sherakama** [comment]
@jbickar ping

**Comment by jbickar** [comment]
> Needed By (Date)
> Sometime in the next few months.

@sherakama, I am _on track!_

**Comment by jbickar** [comment]
2018-08-15-92442-screenshot

**Comment by sherakama** [comment]
Your cert and key is incorrect and it is failing validation.

**Comment by jbickar** [comment]
Whose fault is that, I wonder?

**Comment by sherakama** [comment]
I hard code that configuration in my local.settings.php file as we don't have an environment install task for local dev saml.

File: stanford_ssp.module [hunk 1]
```diff
@@ -1023,8 +1022,8 @@ function stanford_ssp_fetch_from_workgroup_api($group = "") {
   // Add the SSL certificate paths.
   $openssl_defaults = openssl_get_cert_locations();
   $default_path = $openssl_defaults['default_cert_dir'];
-  $cert_filename = variable_get('stanford_ssp_workgroup_api_cert', $default_path . "/stanford-ssp.cert");
-  $key_filename = variable_get('stanford_ssp_workgroup_api_key', $default_path . "/stanford-ssp.key");
```
Comment (jbickar) [inline stanford_ssp.module]: Wait, how is this not broken currently?

Comment (sherakama) [reply stanford_ssp.module]: We set these variables on installation and migration.

Comment (jbickar) [reply stanford_ssp.module]: 😅

**Comment by jbickar** [comment]
```
/app/simplesamlphp/certs/stanford-ssp.key
/app/simplesamlphp/certs/stanford-ssp.cert
```
on Lando? Still no worky.

2018-08-17-121310-screenshot

**Comment by jbickar** [comment]
Doh. Underscore, not hyphen.

**Comment by sherakama** [comment]
I suppose the error messaging could be better. Split out the difference between unable to connect and unable to validate.

**Comment by jbickar** [comment]
Updated comment; still no worky with underscores.

**Comment by sherakama** [comment]
Minions

**Comment by jbickar** [comment]
Per offline conversation, I was using `certs` as the directory, not `cert`. Works fine with:

```
/app/simplesamlphp/cert/stanford_ssp.key
/app/simplesamlphp/cert/stanford_ssp.cert
```
on Lando.

**Comment by jbickar** [comment]
That PR was way more expensive than it should've been.

**Comment by sherakama** [comment]
We discovered a future improvement and something that our users will trip up on as well.
We should better the error messages.

**Comment by jbickar** [comment]
I think I said that in the initial Workgroup PR and you called it "scope creep".

**Comment by sherakama** [comment]
Yeah, new ticket and all that.

**Pull request merged**